XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/UnityInput/MouseController.cs

```diff
@@ -78,7 +78,6 @@ public void Update()
             mouseDelta.y = Input.GetAxis("Mouse X");
             MixedRealityToolkit.InputSystem?.RaiseSourcePositionChanged(InputSource, this, mouseDelta);
             MixedRealityToolkit.InputSystem?.RaiseSourcePoseChanged(InputSource, this, controllerPose);
-            MixedRealityToolkit.InputSystem?.RaiseSourcePositionChanged(InputSource, this, Input.mouseScrollDelta);
```

Comment:
If we've removed this, what is your feeling for how mouse position is managed?

Reply:
I just removed it as an input source change. I think it ought to be gotten via the regular input event changes as the developer will have more control over how this information is presented in the input handler.